tools/plugins/com.liferay.ide.sdk.core/src/com/liferay/ide/sdk/core/SDKUtil.java

```diff
@@ -166,7 +166,9 @@ public static IProject getWorkspaceSDKProject() throws CoreException {
 		IProject[] projects = CoreUtil.getAllProjects();
 
 		for (IProject project : projects) {
-			if (isValidSDKLocation(project.getLocation().toOSString())) {
+			if ((project != null) && (project.getLocation() != null) &&
```

Comment:
Hey @jtydhr88  `CoreUtil.getAllProjects` should never be returning a null project.  can you check that and make that change instead?